api/src/pcapi/core/offers/models.py

```diff
@@ -354,6 +354,8 @@ class Offer(PcObject, Model, ExtraDataMixin, DeactivableMixin, ProvidableMixin):
     #  can be used by PostgreSQL when filtering on the `venueId` column only.
     Index("venueId_idAtProvider_index", venueId, idAtProvider, unique=True)
 
+    Index("offer_expr_idx", ExtraDataMixin.extraData["isbn"].astext)
```

Comment:
```suggestion
    Index("offer_extra_data_isbn_idx", ExtraDataMixin.extraData["isbn"].astext)
```
nitpick: Je propose d'être plus explicite dans le nommage des indexes afin d'éviter les confusions si on ajoute d'autres indexes sur des expressions pour cette table, et/ou si l'on déplace l'ISBN dans une colonne dédiée.